server.js

```diff
@@ -1,21 +1,79 @@
+/* eslint-disable no-undef */
 import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import athletesData from './data/athletes-data.json'
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
+// Example of the object from the Json
+// "affiliateid": "5885",
+//   "affiliatename": "CrossFit Unique",
+//   "age": "45",
+//   "competitorid": "1071438",
+//   "competitorname": "Theresa Ulwahn",
+//   "countryoforigincode": "SE",
+//   "countryoforiginname": "Sweden",
+//   "division": "Women (45-49)",
+//   "divisionid": "4",
+//   "firstname": "Theresa",
+//   "gender": "F",
+//   "height": "1.62",
+//   "is_scaled": "0",
+//   "lastname": "Ulwahn",
+//   "overallrank": "1015",
+//   "overallscore": "6413",
+//   "postcompstatus": "",
+//   "profilepics3key": "0bda5-P1071438_3-184.jpg",
+//   "weight": "61.0"
```

Comment:
Good reference.  Careful though, comments can easily become outdated in real-world situations as new data is added to the database or fields are removed 👍